tests/cn/ghost_bad_ordering.error.c.verify

```diff
@@ -0,0 +1,8 @@
+return code: 1
+tests/cn/ghost_bad_ordering.error.c:5:5: warning: experimental keyword 'cn_ghost' (use of experimental features is discouraged)
+    cn_ghost i32 n; true;
+    ^~~~~~~~ 
+tests/cn/ghost_bad_ordering.error.c:5:5: error: unexpected token after ';' and before 'cn_ghost'
+Please add error message for state 1696 to parsers/c/c_parser_error.messages
+    cn_ghost i32 n; true;
+    ^~~~~~~~ 
```

Comment:
I think I've already said this somewhere but if not, let's make an issue for this error message because it is atrocious and easy to fix (c_parser.error.messages in cerberus repo).